src/utils/gauss_quad.hpp

```diff
@@ -0,0 +1,619 @@
+//  Copyright John Maddock 2015.
+//  Use, modification and distribution are subject to the
+//  Boost Software License, Version 1.0. (See accompanying file
+//  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef GAUSS_QUAD_HPP
+#define GAUSS_QUAD_HPP
```

Comment:
Ditto @brryan 's comments about include guards and namespaces from the other PR

Reply:
fixed